stac_mjx/controller.py

```diff
@@ -538,10 +571,11 @@ def render(
         # render while stepping using mujoco
         with imageio.get_writer(save_path, fps=self.cfg.model.RENDER_FPS) as video:
             for qpos, kps in tqdm(zip(qposes, kp_data)):
-                # Set keypoints
+                # Set keypoints--they're in cartesian space, but since they're attached to the worldbody they're the same as offsets
```

Comment:
The keypoints represented in global cartesian space can be considered offsets relative to the worldbody. I noted this because setting site positions is always relative to the parent body, so one might get confused about why this works with the keypoints' global frame representation.

Reply:
Would it be more clear to say "their the same as global (world) coordinates"? Offsets seems relative to me.